protobuf/api/instance.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+
+package api;
+
+service Instance {
+  rpc Create (CreateInstanceRequest) returns (CreateInstanceResponse) {}
+  rpc Delete (DeleteInstanceRequest) returns (DeleteInstanceResponse) {}
+}
+
+message CreateInstanceRequest {
+  string serviceHash = 1;      // Service's sid or hash.
+  repeated string env = 2;     // Env vars to apply to service's instance on runtime.
+}
+
+message CreateInstanceResponse {
+  string hash = 2;        // Service's instance hash.
+  string serviceHash = 3; // Service's bare hash.
+}
+
+message DeleteInstanceRequest {
+  string hash = 1;      // Instance hash
+}
+
+message DeleteInstanceResponse {
+  string hash = 1;      // Instance hash
```

Comment:
why return instance hash in response if it sent in the request

Reply:
I don't mind much about that, I can remove but the idea is that when we delete a resource we have the id of the resource delete in order to be able to have a full asynchrony application that can on one side send the request and on the other side just receive the result and know how to update a local store or anything.

But again I'm happy to delete it if needed